src/internal.c

```diff
@@ -10228,14 +10230,14 @@ static int GetDtls13RecordHeader(WOLFSSL* ssl, const byte* input,
 #endif /* WOLFSSL_DTLS13 */
 
 #ifdef WOLFSSL_DTLS
-static int GetDtlsRecordHeader(WOLFSSL* ssl, const byte* input,
-    word32* inOutIdx, RecordLayerHeader* rh, word16* size)
+static int GetDtlsRecordHeader(WOLFSSL* ssl, word32* inOutIdx,
+                               RecordLayerHeader* rh, word16* size)
```

Comment:
formatting, on arguments overflowing, 4 spaces on the next line.

Reply:
indentation fixed now